.github/workflows/test-cassandra.yml

```diff
@@ -0,0 +1,36 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'cassandra/**'
+    pull_request:
+        paths:
+            - 'cassandra/**'
+name: "Tests Cassandra"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        services:
+            mongo:
+                image: 'cassandra:latest'
+                ports:
+                    - '27017:27017'
+        strategy:
+            matrix:
+                go-version:
+                    - 1.17.x
```

Comment:
Only 1.19.x and 1.20.x are supported by the go library used to talk to Cassandra